PhysicsTools/SelectorUtils/interface/PFJetIDSelectionFunctor.h

```diff
@@ -480,11 +483,11 @@ class PFJetIDSelectionFunctor : public Selector<pat::Jet>  {
     }else if ( quality_ == TIGHTLEPVETO ) {
       set("CHF", 0.0);
       set("NHF", 0.9);
-      set("CEF", 0.8);
       set("NEF", 0.9);
       set("NCH", 0);
       set("nConstituents", 1);
       if(version_ == WINTER17){
+        set("CEF", 0.8);
 	set("NEF_EC_L",0.02);
```

Comment:
here and below the indentation inside the ```if``` is inconsistent.

Reply:
Done.  This was a space v tabs issue